### PR TITLE
Rename action param to data

### PR DESCRIPTION
**Pull request by timwis**
Per convo in IRC. I separated the commits in case you didn't want one of them; let me know if you'd like me to merge.

There are still some inconsistencies in the docs that need to be updated for 3.0 (re-ordering of params and such) but I left those out of this PR to separate concerns.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,8 +204,8 @@ post the results. `reducers` take data, modify it, and update the internal
 `state`.
 
 Communication of data is done using objects called `actions`. Each `action` has
-any number of properties for data, and a unique `type` that can trigger
-properties on the models.
+any number of properties for `data`, and a unique `actionName` that can trigger
```
Comment (yoshuawuyts) [inline README.md]: Perhaps change this to be somewhere along the lines of:
> Each action consists of a unique `actionName` and an optional payload of `data`, which can be any value.

Perhaps would make it more clear?

**Comment by yoshuawuyts** [comment]
Yup think this looks good

**Comment by timwis** [comment]
Okay, modified the "docs" commit and did a `--force` push on this branch to update it. All systems go?

**Comment by yoshuawuyts** [comment]
All systems :rocket:

**Comment by yoshuawuyts** [comment]
Merging so I can rebase off this :sparkles:

**Pull request merged**